frontend/css/spacing.css

```diff
@@ -0,0 +1,49 @@
+/* Setting up CSS variables */
+:root {
+    --right-padding: 1.5rem;
+    --small-gap: 1.5rem;
+    --large-gap: 2rem;
+    --btm-padding-sml: 0.1rem;
```

Comment:
Just curious, and I know this is still work in progress, but does `btm` stand for "bottom" in this case?  

I would probably go with something like `--spacing-xs`, `--spacing-s`, `--spacing-m` etc for general spacings, sorted in size order. These could then be used both for grid gaps, paddings and margins (regardless of top, bottom, right or left of the element) across the page.